README.md

```diff
@@ -17,19 +17,19 @@ The vizlab wiki at https://github.com/USGS-VIZLAB/vizlab/wiki has help pages on
 
 ## To build/work on a viz
-  First, either clone your fork of the repo, or pull down only the `Dockerfile` and `docker-compose.yml`.  In a terminal, go into the directory containing the `Dockerfile` and `docker-compose.yml`, and run `docker-compose build`.  This builds the docker image using the options in `docker-compose.yml`.  Next, run `docker-compose up`.  This creates and starts the docker container and leaves it running, with Rstudio exposed on port 8787.  Go to your web browser and you can log in to Rstudio at `localhost:8787`.  The username is `rstudio`, and password is set in the `docker-compose.yml`.  Now you can use Rstudio as you would normally, i.e. creating a new project from a Github repo.  Note the container does not contain any of your credentials, for Github or elsewhere.  However, the container already contains the DOI root certificate, so HTTPS will work over the network. (Note that you should **not** upload an **image** containing the DOI cert to a public repo.) Files you save in Rstudio will be contained in a docker volume, and will persist beyond the life of the container (unless you delete the volume, of course).  When you are done, log out of Rstudio and run `docker-compose down` in your terminal to stop the docker container.      
+  First, clone your fork of the repo so you have a local copy of the  `Dockerfile` and `docker-compose.yml`.  In a terminal, go into the directory containing the `Dockerfile` and `docker-compose.yml`, and run `docker-compose build`. Non-USGS people should first delete or comment out lines 10-12 in the `Dockerfile` where the root certificate is retrieved. This builds the docker image using the image name and other options specified in `docker-compose.yml`.  Next, run `docker-compose up`.  This creates and starts the docker container and leaves it running, with Rstudio exposed on port 8787.  Go to your web browser and you can log in to Rstudio at `localhost:8787`.  The username is `rstudio`, and password is `mypass`.  Now you can use Rstudio the same as on a native operating system.  Create a new project from the `File` menu, select Version Control and Git, and enter the URL of your fork of the repository.  Note the container does not contain any of your credentials, for Github or elsewhere.  However, the container already contains the DOI root certificate, so HTTPS will work over the network. (Note that you should **not** upload an **image** containing the DOI cert to a public repo.) Files you save in Rstudio will be contained in a docker volume, and will persist beyond the life of the container (unless you delete the volume, of course).  When you are done, log out of Rstudio and run `docker-compose down` in your terminal to stop the docker container.      
```

Comment:
I like the additions, but now the "This" in sentence 4 is unclear. Could put sentence 3 in parentheses, or move the "run `docker-compose build`: into its own sentence between current sentences 3 and 4, or change the "This" to "Running `docker-compose build`" in sentence 4.